src/server/.env

```diff
@@ -0,0 +1,4 @@
+USER = petyanikleva
+PASSWORD = NQ74bLvDNCZP
+
+PORT = 3030
```

Comment:
good job on using an .env file. :)
But you're not supposed to commit it. I shouldn't be able to see your credentials. You can add .env if your gitignore file

Reply:
Thanks. It has been excluded. @virzhinia-petkova-redshelf